Hello.java

```diff
@@ -0,0 +1,7 @@
+import java.util.Scanner;
+
+public class Hello {
+    public static void main(String[] args){
+        System.out.println("Hello, World!");
+        }
```

Comment:
Vertical alignment of braces does not follow typical coding standard.
Refer to: http://student.cs.appstate.edu/classes/JavaCodingStyle/